assets/component-collection-hero.css

```diff
@@ -9,12 +9,6 @@
 }
 
 @media screen and (min-width: 750px) {
-  .collection-hero.collection-hero--with-image {
-    padding: calc(4rem + var(--page-width-margin)) 0
-      calc(4rem + var(--page-width-margin));
```

Comment:
Could you maybe bring up those use cases in which a visual impact will happen in the PR summary before the gif you have right now ?
This way when doing release notes, it can be caught right away and mentioned in there 👍 
Those notes can be useful to merchants to be aware about what will need attention if they decide to upgrade manually 🙂

Reply:
Thanks. I've drafted a summary in the PR description above, and also [included screenshots below](https://github.com/Shopify/dawn/pull/1790#issuecomment-1179298309).